test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewComponentTests.cs

```diff
@@ -140,6 +140,60 @@ public void ViewComponent_View_WithModelParameter_SetsResultViewWithDefaultViewN
         }
 
         [Fact]
+        public void ViewComponent_View_WithNullModelParameter_SetsResultViewWithDefaultViewNameAndNullModel()
+        {
+            // Arrange
+            var viewComponent = new TestViewComponent();
+
+            // Act
+            var actualResult = viewComponent.View(model: null as object);
+
+            // Assert
+            Assert.IsType<ViewViewComponentResult>(actualResult);
+            Assert.IsType<ViewDataDictionary<object>>(actualResult.ViewData);
+            Assert.NotSame(viewComponent.ViewData, actualResult.ViewData);
+            Assert.Equal(new ViewDataDictionary<object>(viewComponent.ViewData), actualResult.ViewData);
+            Assert.Null(actualResult.ViewData.Model);
+            Assert.Null(actualResult.ViewName);
+        }
+
+        [Fact]
+        public void ViewComponent_View_WithViewNameAndNullModelParameter_SetsResultViewWithViewNameAndNullModel()
+        {
+            // Arrange
+            var viewComponent = new TestViewComponent();
+
+            // Act
+            var actualResult = viewComponent.View("CustomViewName", model: null as object);
```

Comment:
This test is identical to `ViewComponent_View_WithViewNameAndNonObjectNullModelParameter_SetsResultViewWithViewNameAndNullModel`. Use that one (because it avoids `null as object`) but fix `// Arrange`.